src/components/AddBook.js

```diff
@@ -16,7 +16,7 @@ const AddBook = () => {
       author,
       category: 'Fiction',
     };
-    dispatch(addBook(booK));
+    dispatch(createBook(booK));
     setAuthor('');
     setTitle('');
   };
```

Comment:
- [ ] Great job refactoring your addBook functionality to use `CreateAsyncThunk`. However, your add functionality is currently not working so kindly take your time to `fix` this issue as it would help create a better user experience for your users to be able to add books. Below is the return error in the console when I try to create  a new book:
<img width="545" alt="Screenshot 2023-08-22 at 19 06 50" src="https://github.com/As1imwe-Mark/book_store/assets/107192076/f285c177-355a-47f4-8d5a-5810f3d56f33">